centreon/www/class/centreonStatistics.class.php

```diff
@@ -287,4 +302,36 @@ public function getAdditionalData()
 
         return $data;
     }
+
+    /**
+     * @param string $providerType
+     * @return int
+     */
+    private function getAclRelationsByProviderType(string $providerType): int
+    {
+        $query = "SELECT  COUNT(*) AS acl_relation 
+        FROM security_provider_access_group_relation gr
+        INNER JOIN provider_configuration pc on pc.id = gr.provider_configuration_id
+        WHERE pc.type = '$providerType'";
+
+        $result = $this->dbConfig->query($query);
+
+        return $result->fetchColumn();
```

Comment:
[fetchColumn](https://www.php.net/manual/en/pdostatement.fetchcolumn.php) can return a `mixed`.
phpstan should warn about this code.
Please add a cast.
```suggestion
        return (int) $result->fetchColumn();
```